ct/Supabase.sh

```
@@ -0,0 +1,58 @@
+  if [[ ! -d /opt/supabase ]]; then
+    msg_error "No ${APP} Installation Found!"
+    exit
+  fi
+
+  msg_info "Creating Backup"
+  BACKUP_FILE="/opt/supabase/backup_$(date +%Y%m%d_%H%M%S).tar.gz"
+  tar -czf "$BACKUP_FILE" -C /opt/supabase .env 2>/dev/null || true
+  msg_ok "Backup Created"
+
+  msg_info "Updating ${APP}"
+  cd /opt/supabase
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)